libs/ledger-live-common/src/apps/polyfill.ts

```diff
@@ -23,7 +23,7 @@ listCryptoCurrencies(true, true).forEach((a) => {
 
   const dep = findCryptoCurrencyById(a.family);
   if (!dep || !dep.managerAppName) return; // no dep
-
+  if (a.bypassAppDependency) return;
```

Comment:
this new field is **strictly** needed for the **polyfill**? it's a polyfill because it's a limitation, we should eventually replace with a proper API.

therefore I would suggest to NOT introduce a new field on the currency as it's strictly due to this need of polyfill.

i don't see the big picture of why this is required now, but if there are a few exceptions, let's have a local array list in this file instead of having to impact all the currencies definitions. 🙏

Reply:
it is not a "strictly required" field.
as 
defined as

bypassAppDependency?: boolean;

I introduced this field because the bitcoin family coins(altcoins) do not depend on bitcoin any more which is different from the logic before. Any suggestions about it?